backends/platform/android/android.mk

```diff
@@ -9,6 +8,7 @@ GRADLE_FILES = $(shell find $(PATH_DIST)/gradle -type f) $(PATH_DIST)/gradlew $(
 PATH_BUILD = ./android_project
 PATH_BUILD_GRADLE = $(PATH_BUILD)/build.gradle
 PATH_BUILD_ASSETS = $(PATH_BUILD)/assets
+PATH_BUILD_LIB = $(PATH_BUILD)/lib/$(ABI)
 PATH_BUILD_LIBSCUMMVM = $(PATH_BUILD)/lib/$(ABI)/libscummvm.so
```

Comment:
do we need both? is it possible to remove the PATH_BUILD_LIBSCUMMVM

Reply:
It's already used for a couple of targets - going to keep as is.